docs/multi-stage-query/reference.md

```diff
@@ -381,21 +381,14 @@ For more information about clustering, see [Clustering](concepts.md#clustering).
 
 ## Context parameters
 
-In addition to the Druid SQL [context parameters](../querying/sql-query-context.md), the multi-stage query task engine accepts certain context parameters that are specific to it.
-
-Use context parameters alongside your queries to customize the behavior of the query. If you're using the API, include the context parameters in the query context when you submit a query:
-
-```json
-{
-  "query": "SELECT 1 + 1",
-  "context": {
-    "<key>": "<value>",
-    "maxNumTasks": 3
-  }
-}
-```
+Along with the standard Druid SQL [context parameters](../querying/sql-query-context.md), the multi-stage query task engine also supports its own set of specific context parameters. These parameters can be used with your queries to tailor how they are executed. 
+
+For detailed instructions on configuring query context parameters, refer to the [query context parameters](../querying/query-context.md) documentation.
+
+Beyond that, you can also define context parameters using `SET` statements directly within your SQL query. This method is especially useful for quick, inline parameter configuration. 
+For how to using `SET` command, See [SET Command](../querying/sql.md#set).
 
-If you're using the web console, you can specify the context parameters through various UI options.
+Note: While the examples above use the `SELECT` statement, you can also specify context parameters in Druid SQL queries that use `INSERT` or `REPLACE`.
```

Comment:
Move this before "For detailed instruction" since it's more important. Suggested revision:

```
You can specify the context parameters in SELECT, INSERT, or REPLACE statements.
```